code/game/machinery/bots/secbot.dm

```diff
@@ -384,6 +384,22 @@ Auto Patrol: []"},
 		target = user
 		mode = BOT_HUNT
 
+/obj/machinery/bot/secbot/Crossed(atom/movable/AM)
+	if(ismob(AM) && target)
+		var/mob/living/carbon/C = AM
+		if(!istype(C) || !C || in_range(src, target))

```

Comment:
The in_range check is for if Beepsky is next to his target, not if he's next to the person who crossed it. This is so that people don't fall over just by walking over Beepsky while he's standing still and ziptying a guy.